common/pom.xml

```diff
@@ -1,40 +1,48 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-	<modelVersion>4.0.0</modelVersion>
-
-	<parent>
-		<groupId>org.openthinclient</groupId>
-		<artifactId>manager</artifactId>
-		<version>2.0.0-SNAPSHOT</version>
-		<relativePath>..</relativePath>
-	</parent>
-
-	<name>openthinclient.org common components</name>
-	<artifactId>manager-common</artifactId>
-	<packaging>jar</packaging>
-
-	<dependencies>
-		<dependency>
-			<groupId>castor</groupId>
-			<artifactId>castor</artifactId>
-		</dependency>
-
-		<dependency>
-			<groupId>net.sf.ehcache</groupId>
-			<artifactId>ehcache</artifactId>
-		</dependency>
-
-		<dependency>
-			<groupId>org.openthinclient</groupId>
-			<artifactId>manager-service-apacheds</artifactId>
-			<version>2.0.0-SNAPSHOT</version>
-			<scope>test</scope>
-			<type>jar</type>
-		</dependency>
-		
-		<dependency>
-			<groupId>log4j</groupId>
-			<artifactId>log4j</artifactId>
-		</dependency>
-	</dependencies>
-</project>
\ No newline at end of file
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+	<groupId>org.openthinclient</groupId>
+	<artifactId>manager</artifactId>
+	<version>2.0.0-SNAPSHOT</version>
+	<relativePath>..</relativePath>
+    </parent>
+
+    <name>openthinclient.org common components</name>
+    <artifactId>manager-common</artifactId>
+    <packaging>jar</packaging>
+
+    <dependencies>
+	<dependency>
+	    <groupId>castor</groupId>
+	    <artifactId>castor</artifactId>
+	</dependency>
+
+	<dependency>
+	    <groupId>net.sf.ehcache</groupId>
+	    <artifactId>ehcache</artifactId>
+	</dependency>
+
+	<dependency>
+	    <groupId>org.openthinclient</groupId>
+	    <artifactId>manager-service-apacheds</artifactId>
+	    <version>2.0.0-SNAPSHOT</version>
+	    <scope>test</scope>
+	    <type>jar</type>
+	</dependency>
+
+	<dependency>
+	    <groupId>log4j</groupId>
+	    <artifactId>log4j</artifactId>
+	</dependency>
```

Comment:
As far as possible, we should not rely on a particular logging framework. Instead use the SLF4J Logging API.

Constructing a logger using SLF4J would be something like the following:

```java
Logger LOG = LoggerFactory.getLogger(SomeClass.class);
```